app/models/deploy_queue.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class DeployQueue
```

Comment:
better name would be good since we already have a queue ... DeployStaggerer or so ...